less/home-page.less

```
@@ -58,18 +81,33 @@
       }
 
       .btn {
```

We don't need that button anymore. Change that to an a tag.

```suggestion
      a {
```